tests/integration-tests/tests/cloudwatch_logging/test_cloudwatch_logging/test_cloudwatch_logging/pcluster.config.ini

```diff
@@ -0,0 +1,26 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = {{ queue_size }}
+max_queue_size = {{ queue_size }}
```

Comment:
Why do you need `max_queue_size `?

Reply:
I don't.

Reply:
please remove

Reply:
I did. I just haven't pushed it yet. I was going to incorporate the necessary changes for all of the feedback before pushing.